lib/qm-dsp/dsp/keydetection/GetKeyMode.cpp

```diff
@@ -103,8 +102,19 @@ GetKeyMode::GetKeyMode( int sampleRate, float tuningFrequency,
     
     m_MajCorr = new double[m_BPO];
     m_MinCorr = new double[m_BPO];
-    m_Keys  = new double[2*m_BPO];
     
+    m_MajProfileNorm = new double[m_BPO];

```

Comment:
m_BPO is just the constant 36. Static allocation would work for all those arrays as well as a single dynamic allocation with partitioning/slicing.